Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRSupport.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using UnityEngine.XR;
```

Comment:
Wondering, do we need #ifs around all the XR stuff like in `TrackedPoseDriver`?

```
#if ENABLE_VR && UNITY_INPUT_SYSTEM_ENABLE_XR
```

Semi-related, what's the status of pulling out all the XR things into a separate package?

Reply:
I think you talked to Matt about this, this morning.
The plan was looking at 2020.1, 
XRTemplateBuilder would still be in the InputSystem package, but the individual profiles would be located in each package.

Reply:
I don't think we _need_ those ifs around it,  as all the device setup code will still compile on a platform with no VR support. But it might make sense to just avoid unnecessary code?

Reply:
I think the reference I'm using is in com.unity.modules.xr, so I'll wrap it up in UNITY_INPUT_SYSTEM_ENABLE_XR